### PR TITLE
sed curl to cover uses inside install.sh

### PR DESCRIPTION
**Pull request by bobzilladev**
This sed replaces the curl in the install.sh before handing to bash.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,7 +13,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: cachix/install-nix-action@v18
-      - run: curl --retry 5 --retry-all-errors -sfL https://direnv.net/install.sh | bash
+      - run: |
+          curl -v --retry 5 --retry-all-errors -sfL https://direnv.net/install.sh | sed 's/curl/curl -v --retry 5 --retry-all-errors --header "authorization: Bearer ${{ secrets.GITHUB_TOKEN }}"/g' | bash
```
Comment (jrobsonchase) [inline .github/workflows/ci.yml:17]: Oof. Might want to toss a [yaml anchor](https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/) on this first one, and then use that for the other jobs. Save us some copy-pasta.

**Review by jrobsonchase — APPROVED** [review]
Approach looks good :+1:

This action has been *real* frustrating :confused:

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bobzilladev) [reply .github/workflows/ci.yml:17]: if i can prove this works, i've had a lot of false hope :)

Comment (bobzilladev) [reply .github/workflows/ci.yml]: github says nope:
`The workflow is not valid. .github/workflows/ci.yml: Anchors are not currently supported. Remove the anchor 'install-direnv'`

Comment (bobzilladev) [reply .github/workflows/ci.yml]: I moved to multiline to make it more readable. Haven't had a failure after many runs so going to merge this and 🤞

**Pull request merged**